PROJECT

```diff
@@ -1,3 +1,22 @@
-version: "1"
-domain: k8s.io
+version: "2"
+domain: x-k8s.io
```

Comment:
why do we need to change the domain?

Reply:
(don't think it's wrong, just curious)

Reply:
The crds (https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/config/crd/bases/infrastructure.cluster.x-k8s.io_azureclusters.yaml) are already using the `x-k8s`